gcc/jit/jit-recording.cc

```diff
@@ -4210,6 +4223,40 @@ recording::function::new_block (const char *name)
 void
 recording::function::write_to_dump (dump &d)
 {
+  for (auto attr: m_attributes)
```

Comment:
Do we have the `write_to_dump()` for variable attributes?

Reply:
For now it's not really useful but later on, as we expand the support, would be nice.

Reply:
Done.